WordPress/Classes/ViewRelated/Tools/Time Zone/TimeZoneSelectorViewController.swift

```diff
@@ -65,22 +65,79 @@ struct TimeZoneSelectorViewModel: Observable {
         })
     }
 
+    private let timeZoneOffsetFormatter = DateFormatter()
+
+    private let timeAtTimeZoneFormatter = DateFormatter()
+
+    private let currentDate = Date()
```

Comment:
Splitting the initialization between these lines and the initializer makes the code less clear. Also, since the creation of `DateFormatter` is indeed expensive, why not put the instantiation of them behind the feature flag as well?
How can we change this code to be more clear, and, potentially, also more testable?

Reply:
That's a good idea and I've taken your suggestion and run with it by encapsulating that logic in `TimeZoneFormatter` and adding accompanying unit tests. Thanks!